packages/@aws-cdk/aws-codebuild/README.md

```diff
@@ -213,7 +213,7 @@ The CodeBuild library supports both Linux and Windows images via the
 `LinuxBuildImage` and `WindowsBuildImage` classes, respectively.
 
 You can specify one of the predefined Windows/Linux images by using one
-of the constants such as `WindowsBuildImage.WINDOWS_BASE_2_0` or
+of the constants such as `WindowsBuildImage.WINDOWS_BASE_2_0`, or `WindowsBuildImage.WIN_SERVER_CORE_2019_BASE` or
```

Comment:
This documentation is now out of date.

Please mention how to use the new Windows type somewhere in the ReadMe, preferably with code example(s).

Reply:
I have added details on how to pass the `WindowsImageType` optional parameter. I have also fixed a mistaken in the previous documentation: specifically, from the [LinuxBuildImage](https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-codebuild.LinuxBuildImage.html) and [WindowsBuildImage](https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-codebuild.WindowsBuildImage.html) documentation pages I see that [fromCodeBuildImageId](https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-codebuild.LinuxBuildImage.html#static-from-wbr-code-wbr-build-wbr-image-wbr-idid) is present only in `LinuxBuildImage`.